src/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './CommentList.scss';
+
+import PropTypes from 'prop-types';
+
+export const CommentList = ({ comments }) => (
+  <section className="comments">
+    <h3 className="comments__title">
+      Comments:
+    </h3>
+
+    {comments.map(comment => (
+      <article className="comments__comment comment" key={comment.id}>
+        <p className="comment__name">
+          {comment.name}
+        </p>
+
+        <p className="comment__body">
+          {comment.body}
+        </p>
+
+        <p className="comment__email">
+          {comment.email}
+        </p>
+      </article>
```

Comment:
<img width="874" alt="image" src="https://user-images.githubusercontent.com/38065883/91963522-a1553200-ed16-11ea-8b74-9d38a4a04fdd.png">
Where is the Comment component?